packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { ThemeProvider } from 'styled-components';
+import defaultTheme from 'es-components-via-theme';
+import TabList from './TabList';
+import Tab from './Tab';
+import { some } from 'lodash';
+
+const TabWrapper = styled.div`
+  display: flex;
+  border-bottom: 1px solid ${props => props.theme.colors.gray4};
+`;
+
+const TabFormatter = styled.div`
+  display: flex;
+  flex-direction: column;
+  font-size: inherit;
+  @media (max-width: ${props => props.theme.screenSize.desktop}) {
+    width: 100%;
+  }
+  @media (min-width: ${props => props.theme.screenSize.desktop}) {
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+`;
+
+const TabContent = styled('div')``;
```

Comment:
If this isn't getting any styles applied, let's remove this and just use a div.

Reply:
Yep I was able to reduce it to nothing, so I can use a div.